program/src/instruction.rs

```diff
@@ -1,14 +1,17 @@
 //! Program instruction types.
 
-use solana_program::{
-    instruction::{AccountMeta, Instruction},
-    program_error::ProgramError,
-    pubkey::Pubkey,
-    system_program,
+use {
+    shank::ShankInstruction,
+    solana_program::{
+        instruction::{AccountMeta, Instruction},
+        program_error::ProgramError,
+        pubkey::Pubkey,
+        system_program,
+    },
 };
 
 /// Instructions supported by the Paladin Rewards program.
-#[derive(Clone, Copy, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, PartialEq, ShankInstruction)]
```

Comment:
Usually is more readable to include a `#[rustfmt::skip]` to the `enum` so shank annotations do not get formatted – each account stay on a single line.

Reply:
I agree, I also hate the variance in formatting. I see your `#[rustfmt::skip]` and raise you vertical instead of horizontal (I like the vertical better). 😎